src/js/info.js

```diff
@@ -42,15 +35,15 @@ class VideoInfo {
 
 	update () {
 		if (this.brandEl) {
-			this.brandEl.textContent = formatBrand(this.video.videoData.tags);
+			this.brandEl.textContent = this.video.videoData.brand.name;
 		}
 
 		if (this.titleEl) {
 			this.titleEl.textContent = this.video.videoData.name;
```

Comment:
Just checking if the response from Next media API is as close to the agreed CAPI model - so in future we can maybe drop the modelling step when we switch away from the ES data source.